trempy/shared/shared_auxiliary.py

```diff
@@ -19,45 +20,52 @@
 
 
 def criterion_function(df, questions, cutoffs, paras_obj, version, sds, **version_specific):
-    """Calculate the likelihood of the observed sample.
-
-    sds: standard deviations for each question.
-    model_obj: the ModelCls object
-    version: nonstationary or scaled_archimedean
-    """
-    # Distribute parameters
-    m_optimal = get_optimal_compensations(version=version, paras_obj=paras_obj, questions=questions,
-                                          **version_specific)
-
-    contribs = []
-    for i, q in enumerate(questions):
```

Comment:
beautful improvements using pandas properly :1st_place_medal:  I wrote this before digging in for a couple of days practicing the library